assets/templates/dataset-filter/list-selector.tmpl

```diff
@@ -50,16 +50,16 @@
                                         </div>
 								</fieldset>
 								<div class="margin-top js-hidden">
-									<input type="submit" value="Add selected filters" class="btn btn--secondary btn--focus font-weight-700 font-size--17 text-wrap">
+									<input type="submit" value="Add selected filters" class="btn btn--secondary btn--focus font-weight-700 font-size--18 line-height--32 text-wrap">
```

Comment:
Like other comments above, we should be able to just remove any `font-size--18` class here as we have set it in `adjust-font-size--18` above